whylabs_toolkit/monitor/diagnoser/helpers/utils.py

```diff
@@ -0,0 +1,68 @@
+import os
+from typing import List, Optional
+
+from whylabs_client.api.monitor_diagnostics_api import MonitorDiagnosticsApi
+
+from whylabs_toolkit.helpers.client import create_client
+from whylabs_toolkit.helpers.config import Config
+from whylabs_toolkit.monitor.models import SegmentTag
+
+
+def get_monitor_diagnostics_api(config: Config = Config()) -> MonitorDiagnosticsApi:
+    """
+    Get the monitor diagnostics API, which is used to interact with the WhyLabs Monitor Diagnostics service
+    to diagnose noisy monitors.
+    :param config:
+    :return:
+    """
+    return MonitorDiagnosticsApi(api_client=create_client(config=config))
+
+
+def env_setup(
```

Comment:
most of this logic is also covered on: https://github.com/whylabs/whylabs-toolkit/blob/mainline/whylabs_toolkit/helpers/config.py

Reply:
I think the logic in Config is part of why I created env_setup - because it requires one set of env vars; whylogs requires another and I wanted to define them once and pass them in programmatically!  We could promote env setup into helpers if you think that makes sense